wls-auth-service/src/main/resources/application-test.yml

```
@@ -14,3 +14,11 @@ spring:
     properties:
       hibernate:
         format_sql: true
+
+service:
+  config:
+    oauth2:
+      jwk:
+        rsa:
+          init:
+            seed: change_me
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)